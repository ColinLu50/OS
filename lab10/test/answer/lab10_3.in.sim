12947
1011
1427
809
1411
801
