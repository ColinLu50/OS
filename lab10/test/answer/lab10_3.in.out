FCFS
613 -> 692 -> 377 -> 631 -> 366 -> 691 -> 422 -> 788 -> 417 -> 110 -> 669 -> 171 -> 814 -> 406 -> 551 -> 349 -> 551 -> 723 -> 734 -> 730 -> 596 -> 730 -> 571 -> 611 -> 690 -> 213 -> 785 -> 124 -> 164 -> 345 -> 515 -> 214 -> 220 -> 8 -> 751 -> 212 -> 618 -> 130 -> 115 -> 388 -> 104 -> 635 -> 511 -> 214 -> 441 -> 568 -> 236
12947
SSTF
613 -> 611 -> 618 -> 631 -> 635 -> 669 -> 690 -> 691 -> 692 -> 723 -> 730 -> 730 -> 734 -> 751 -> 785 -> 788 -> 814 -> 596 -> 571 -> 568 -> 551 -> 551 -> 515 -> 511 -> 441 -> 422 -> 417 -> 406 -> 388 -> 377 -> 366 -> 349 -> 345 -> 236 -> 220 -> 214 -> 214 -> 213 -> 212 -> 171 -> 164 -> 130 -> 124 -> 115 -> 110 -> 104 -> 8
1011
SCAN
613 -> 611 -> 596 -> 571 -> 568 -> 551 -> 515 -> 511 -> 441 -> 422 -> 417 -> 406 -> 388 -> 377 -> 366 -> 349 -> 345 -> 236 -> 220 -> 214 -> 213 -> 212 -> 171 -> 164 -> 130 -> 124 -> 115 -> 110 -> 104 -> 8 -> 0 -> 618 -> 631 -> 635 -> 669 -> 690 -> 691 -> 692 -> 723 -> 730 -> 734 -> 751 -> 785 -> 788 -> 814
1427
C-SCAN
613 -> 611 -> 596 -> 571 -> 568 -> 551 -> 515 -> 511 -> 441 -> 422 -> 417 -> 406 -> 388 -> 377 -> 366 -> 349 -> 345 -> 236 -> 220 -> 214 -> 213 -> 212 -> 171 -> 164 -> 130 -> 124 -> 115 -> 110 -> 104 -> 8 -> 0 -> 814 -> 814 -> 788 -> 785 -> 751 -> 734 -> 730 -> 723 -> 692 -> 691 -> 690 -> 669 -> 635 -> 631 -> 618
809
LOOK
613 -> 611 -> 596 -> 571 -> 568 -> 551 -> 515 -> 511 -> 441 -> 422 -> 417 -> 406 -> 388 -> 377 -> 366 -> 349 -> 345 -> 236 -> 220 -> 214 -> 213 -> 212 -> 171 -> 164 -> 130 -> 124 -> 115 -> 110 -> 104 -> 8 -> 618 -> 631 -> 635 -> 669 -> 690 -> 691 -> 692 -> 723 -> 730 -> 734 -> 751 -> 785 -> 788 -> 814
1411
C-LOOK
613 -> 611 -> 596 -> 571 -> 568 -> 551 -> 515 -> 511 -> 441 -> 422 -> 417 -> 406 -> 388 -> 377 -> 366 -> 349 -> 345 -> 236 -> 220 -> 214 -> 213 -> 212 -> 171 -> 164 -> 130 -> 124 -> 115 -> 110 -> 104 -> 8 -> 814 -> 788 -> 785 -> 751 -> 734 -> 730 -> 723 -> 692 -> 691 -> 690 -> 669 -> 635 -> 631 -> 618
801
